build.sbt

```
@@ -3,7 +3,7 @@ organization := "io.joern"
 
 scalaVersion := "2.13.6"
 
-val cpgVersion       = "1.3.380"
+val cpgVersion       = "1.3.390"
```

```suggestion
val cpgVersion       = "1.3.391"
```
Necessary for the upcoming try-catch PR